paddle_billing/Resources/Subscriptions/Operations/Update/UpdateBillingDetails.py

```diff
@@ -0,0 +1,13 @@
+from __future__ import annotations
+from dataclasses import dataclass
+
+from paddle_billing.Entities.Shared.Duration import Duration
+from paddle_billing.Undefined import Undefined
+
+
+@dataclass
+class UpdateBillingDetails:
+    payment_terms: Duration | Undefined = Undefined()
+    enable_checkout: bool | Undefined = Undefined()
+    purchase_order_number: str | Undefined = Undefined()
+    additional_information: str | Undefined = Undefined()
```

Comment:
Just checking, spec is showing this field as nullable, so I think we need this...
```suggestion
    additional_information: str | None | Undefined = Undefined()
```

Reply:
Fixed in 9c8222f